### PR TITLE
Run regresscheck-shared on PG13

### PR DESCRIPTION
**Pull request by svenklemm**
This patch enables the regresscheck-shared testsuite to run on PG13

### CODE REVIEW

File: tsl/test/sql/gapfill-13.sql [hunk 1]
```diff
@@ -0,0 +1,1653 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+\set EXPLAIN 'EXPLAIN (COSTS OFF)'
+
+CREATE TABLE gapfill_plan_test(time timestamptz NOT NULL, value float);
+SELECT table_name FROM create_hypertable('gapfill_plan_test','time',chunk_time_interval=>'4 weeks'::interval);
```
Comment (mkindahl) [inline tsl/test/sql/gapfill-13.sql:8]: Don't we need to test this for all versions? It seems to be only added for PG13.

File: tsl/test/sql/remote_copy-13.sql [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+\c :TEST_DBNAME :ROLE_CLUSTER_SUPERUSER;
+
+-- Cleanup from other potential tests that created these databases
+SET client_min_messages TO ERROR;
+DROP DATABASE IF EXISTS data_node_1;
+DROP DATABASE IF EXISTS data_node_2;
+DROP DATABASE IF EXISTS data_node_3;
+SET client_min_messages TO NOTICE;
+
```
[the remark this answers is not in the document]
Comment (mkindahl) [reply tsl/test/sql/remote_copy-13.sql:13]: Also, I do not see it added to any `CMakeLists.txt` file.

**Review by mkindahl — COMMENTED** [review]
Just a few things I noticed:
- There are two tests added that are only for PG13. Don't we want to run those tests for PG11 and PG12 as well?
- I don't see the tests added to any `CMakeLists.txt` file.

File: tsl/test/sql/gapfill-13.sql (hunk 1 above)
Comment (svenklemm) [reply tsl/test/sql/gapfill-13.sql]: Hmm those files are from different testsuite not sure why they were in this PR too. Removed now.

**Review by mkindahl — APPROVED** [review]

**Review by gayyappan — APPROVED** [review]

**Pull request merged**